polling_stations/apps/data_collection/management/commands/import_wealden.py

```diff
@@ -3,17 +3,28 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E07000065"
-    addresses_name = (
-        "europarl.2019-05-23/Version 1/Democracy_Club__23May2019Wealdon.tsv"
-    )
-    stations_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019Wealdon.tsv"
-    elections = ["europarl.2019-05-23"]
+    addresses_name = "parl.2019-12-12/Version 1/merged.tsv"
+    stations_name = "parl.2019-12-12/Version 1/merged.tsv"
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
+    allow_station_point_from_postcode = False
 
     def address_record_to_dict(self, record):
         rec = super().address_record_to_dict(record)
         uprn = record.property_urn.strip().lstrip("0")
 
+        if record.addressline6 in [
+            "BN27 1DQ",
+            "TN210BA",
```

Comment:
this should probably have a space in it to match on `addressline6` in an Xpress export